modules/ROOT/nav.adoc

```diff
@@ -225,17 +225,17 @@
 ** xref:manage:io-optimization.adoc[]
 ** xref:manage:raft-group-reconfiguration.adoc[Raft Group Reconfiguration]
 * xref:reference:index.adoc[Reference]
+** xref:reference:properties/index.adoc[]
+*** xref:reference:properties/broker-properties.adoc[]
+*** xref:reference:properties/cluster-properties.adoc[]
+*** xref:reference:properties/topic-properties.adoc[]
+*** xref:reference:properties/object-storage-properties.adoc[]
```

Comment:
Since object storage properties are cluster properties, suggest that it goes under the Cluster Properties file (before Topic Properties).